pkg/system/path_deprecated.go

```diff
@@ -0,0 +1,18 @@
+package system
+
+const defaultUnixPathEnv = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+// DefaultPathEnv is unix style list of directories to search for
+// executables. Each directory is separated from the next by a colon
+// ':' character .
+// For Windows containers, an empty string is returned as the default
+// path will be set by the container, and Docker has no context of what the
+// default path should be.
+//
+// Deprecated: use oci.DefaultPathEnv
+func DefaultPathEnv(os string) string {
+	if os == "windows" {
```

Comment:
Can this be a call to `oci.DefaultPathEnv`?

Reply:
Doing so would add `oci` (including containerd as indirect) as dependency for a `pkg/`, which wasn't desirable, and would fail in CI: https://github.com/moby/moby/blob/3b84630e147067a360b17e17644020938353d8a4/hack/validate/pkg-imports#L14-L20

I couldn't find external consumers of this function, so perhaps we could've skipped the "deprecated" altogether, but we should be able to remove this file after it's been in a release.